src/ert/ensemble_evaluator/builder/_legacy.py

```diff
@@ -172,7 +169,7 @@ async def _evaluate_inner(
         result = CloudEvent(
             {
                 "type": identifiers.EVTYPE_ENSEMBLE_STOPPED,
-                "source": f"/ert/ee/{self._ee_id}/ensemble",
```

Comment:
Have yet to make sure that this event is handled properly when the `ensemble` suffix is removed.

Reply:
Did not find any reasons for this suffix to change the behaviour anywhere